shell.nix

```diff
@@ -1 +1 @@
-(import ./release.nix).cardano-shell.env
+(import ./release.nix).cardano-shell
```

Comment:
worky worky?

Reply:
No worky... will put it back to WIP until we can sort this out.